setup.py

```diff
@@ -41,7 +41,8 @@ def load_test_suite():
     test_suite='setup.load_test_suite',
     packages=find_packages(),
     install_requires=['spglib', 'numpy', 'scipy',
-                      'h5py', 'phonopy>=2.1.3', 'matplotlib', 'seekpath'],
+                      'h5py', 'phonopy>=2.1.3', 'matplotlib', 'seekpath',
+                      'castepxbin@git+https://github.com/zhubonan/castepxbin.git@v0.0.1'],
```

Comment:
Can we move this into "extras" rather than have it as a mandatory requirement? Maybe we can have a 'castep' option for all castep-related extras.

Reply:
I have updated it - now `castepxbin` is release on pypi. I have pinned it to the `0.1.0` version for now.